test/devicecontroller/DefaultDeviceController.test.ts

```diff
@@ -2408,48 +2415,182 @@ describe('DefaultDeviceController', () => {
 
   describe('preview', () => {
     let element: HTMLVideoElement;
+    let spy1: sinon.SinonSpy;
```

Comment:
can we actually name spy1,2,3 to something specific like gUM?

Reply:
Done